src/AppBundle/Entity/Component.php

```diff
@@ -0,0 +1,282 @@
+<?php
```

Comment:
Please check to add additional validations, for example NotNull or NotBlank and so on